requirements/edx/base.in

```diff
@@ -43,7 +43,6 @@ defusedxml
 Django                              # Web application framework
 django-appconf
 django-celery-results               # Only used for the CacheBackend for celery results
-django-cookies-samesite             # Middleware which allows SameSite=None flag for session and csrf cookies in Django<3.0.5
```

Comment:
This isn't required in Django 3.2 as it is already included in Django now from version 3.1

[BOM-2752](https://openedx.atlassian.net/browse/BOM-2752)